### PR TITLE
Kp update

### LINKED ISSUE #1: Incorrect Explanation of Specificity
I am fairly confident Specificity is defined wrong. The pop up describes it as, “the probability of someone who does not have the disease testing positive for it”. However, that would mean slider default of .99 would be .99 probability that there would be .99 false positive chance. I believe it should be, "the probability of someone who does not have the disease testing negative for it."

### LINKED ISSUE #2: Simple Typo on button
On the prerequisites page there is a button labeled "Go to prerequisites" when really it should say, "Go to Explore" or something similar.

### PR DESCRIPTION
**Pull request by kqp5476**
**Summary of changes:** 
Reordered top right buttons (info, comments, and home); 
Added corresponding URLs to top right buttons; 
Left sidebar menu (added references page); 
Included all appropriate info on References page; 
Revised "Acknowledgements" on Overview page; 
More specific text on buttons on Overview and Prereq pages; 
Matched layout on Challenge page to that of style guide; 
Improved readability of calculation output on Challenge page; 
Inserted boastUtils to standardize fonts, colors, copyright info, etc.

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+              wellPanel(
+                bsButton(
+                  inputId = "ques", 
+                  label = "New Challenge"),
+                br(), 
+                br(),
+                textOutput("question"),
+                br(), 
+                bsButton(
+                  inputId = "show_ans",
```
Comment (shravanisamala) [inline app.R]: Might need to add style, class, and color too

File: ui.R [hunk 2]
```diff
@@ -10,117 +10,151 @@ shinyUI(fluidPage(
...
-                                  column(12, align="center",
-                                         uiOutput('calculation')),
-                                  br(),
-                                  column(3, 
-                                         checkboxInput('pop_result', 'Show Theoretical Result', value = TRUE))
-                                )
-                                )))
-                              )
-                            )
-                            )))
```
Comment (shravanisamala) [inline ui.R:126]: Github says there should be a newline here

File: ui.R [hunk 3]
```diff
@@ -10,117 +10,151 @@ shinyUI(fluidPage(
...
+                  uiOutput('calculation'), 
+                  checkboxInput(
+                    inputId = 'pop_result', 
+                    label = 'Show Theoretical Result', 
+                    value = TRUE)
+                )
+              )
+      )
+    ))
+))
```
Comment (shravanisamala) [inline ui.R]: Github says there should be a newline here

**Review by shravanisamala — DISMISSED** [review]
Left some comments in the code!

File: app.R (hunk 1 above)
Comment (shravanisamala) [reply app.R]: Also I think you need to add a Description file because the style guide doesn't have CODEOWNERS anymore!

**Comment by neilhatfield** [comment]
@kqp5476 Is there a DESCRIPTION file on this branch that I'm just not seeing?

**Pull request merged**